puppet/modules/quickstack/manifests/pacemaker/ceilometer.pp

```diff
@@ -27,6 +29,25 @@
       $_ensure = undef
     }
 
+    if $coordination_backend == 'redis' {
+      $_initial_master = $backend_ips[0]
+      $coordination_url = "redis://${initial_master}:${coordination_backend_port}"
```

Comment:
Minor nit, please make this $_coordination_url for consistency with other internal variables